### ISSUE TITLE
Is this yet another dead GitHub project or dead author?

### ISSUE THREAD

**Issue by ghost** [ankitpopli1891/django-autotranslate#40]
Is this yet another dead GitHub project or author?
The pull requests are not processed, has somebody more information about what is going on?

**Comment by melmakioui** [comment]
> Is this yet another dead GitHub project or author? The pull requests are not processed, has somebody more information about what is going on?

Good Q, I wonder about that too.

**Comment by ankitpopli1891** [comment]
haha, not a dead author yet, but definitely not maintaining the project. I'll try to make the necessary changes, and update the README that this is not maintained anymore.

**Comment by bckohan** [comment]
@ankitpopli1891 since you no longer maintain it, would you be willing to hand over the keys to the name on pypi? I'd be happy to fork and maintain.

**Comment by ankitpopli1891** [comment]
> [@ankitpopli1891](https://github.com/ankitpopli1891) since you no longer maintain it, would you be willing to hand over the keys to the name on pypi? I'd be happy to fork and maintain.

Yep, why not. I'm happy to add you as a collaborator here, and on pypi as well.

**Comment by bckohan** [comment]
> > [@ankitpopli1891](https://github.com/ankitpopli1891) since you no longer maintain it, would you be willing to hand over the keys to the name on pypi? I'd be happy to fork and maintain.
> 
> Yep, why not. I'm happy to add you as a collaborator here, and on pypi as well.

https://pypi.org/user/bckohan/

**Comment by bckohan** [comment]
I'll be issuing maintenance updates.
